pkg/volume/cinder/attacher.go

```diff
@@ -240,7 +240,6 @@ func (attacher *cinderDiskAttacher) WaitForAttach(spec *volume.Spec, devicePath
 	defer timer.Stop()
 
 	for {
-		probeAttachedVolume()
```

Comment:
Where is this done instead?

Reply:
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/cinder/attacher.go#L243
and
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/cinder/attacher.go#L247
 probeAttachedVolume() 2 times per second, but only need to call it one time.